src/scripts/main.js

```diff
@@ -22,6 +22,19 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
```

Comment:
The check on line 32 is unnecessary. The 'for...in' loop on line 27 will not run if 'data[key]' is an empty object, so there's no need to check if 'data[key]' has any keys before calling 'createTree'. You can safely remove lines 32 and 34.